src/components/constants/Items.ts

```diff
@@ -408,6 +408,226 @@ export class Items {
         "Town Map": {
             "effect": "Shows your position in the Pokemon World",
             "category": "Key"
-        }
+        },
+        "HM01": {
+            "effect": "Cut",
```

Comment:
As context, this item schema was made a while ago as a temporary holdover until real item usage is enabled. Don't feel pressured to stick with it at all.